packages/router/src/directives/router_link_active.ts

```diff
@@ -99,6 +108,16 @@ export class RouterLinkActive implements OnChanges, OnDestroy, AfterContentInit
    */
   @Input() routerLinkActiveOptions: {exact: boolean}|IsActiveMatchOptions = {exact: false};
 
+
+  /**
+   * Aria-current attribute to apply when the router link is active.
+   *
+   * Possible values: `'page'` | `'step'` | `'location'` | `'date'` | `'time'` | `true` | `false`.
+   *
+   * @see {@link https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current}
+   */
+  @Input() ariaCurrent?: 'page'|'step'|'location'|'date'|'time'|true|false;
```

Comment:
```suggestion
  @Input() ariaCurrentWhenActive?: 'page'|'step'|'location'|'date'|'time'|true|false;
```

I think a small increase in the verbosity of the input is worthwhile since it's then a _lot_ more clear what this is by clarifying two things:

* It belongs to the `routerLink/routerLinkActive` directive
* It applies the given aria-current value _when the routerLink is active_

Reply:
yeah you're totally right :slightly_smiling_face::+1: